pkg/streamingpromql/engine_test.go

```diff
@@ -1451,26 +1455,26 @@ func TestMemoryConsumptionLimit_SingleQueries(t *testing.T) {
 
 			// Each series has five samples, which will be rounded up to 8 (the nearest power of two) by the bucketed pool,
 			// and we have five series and each of the series has SeriesMetadata.
-			rangeQueryExpectedPeak: 5*8*types.FPointSize + 8*types.SeriesMetadataSize,
+			rangeQueryExpectedPeak: 5*8*types.FPointSize + 8*types.SeriesMetadataSize + 5*uint64(limiter.StringSize)*uint64(nameLength+len("some_metric")+len("idx")+len("1")),
```

Comment:
Once you've added a the size method you mention below to `labels.Labels`, I would expect these tests to use that as well.